### PR TITLE
Update changelog for v9.0.5

### PR DESCRIPTION
**Pull request by danjm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 ## Current Develop Branch
 
 ## 9.0.5 Mon Feb 08 2021
+- [#10278](https://github.com/MetaMask/metamask-extension/pull/10278): Allow editing transaction amount after clicking max
+- [#10214](https://github.com/MetaMask/metamask-extension/pull/10214): Standardize size, shape and color of network color indicators
+- [#10298](https://github.com/MetaMask/metamask-extension/pull/10298): Use network primary currency instead of always defaulting to ETH in the confirm approve screen
+- [#10300](https://github.com/MetaMask/metamask-extension/pull/10300): Add origin to signature request confirmation page
+- [#10296](https://github.com/MetaMask/metamask-extension/pull/10296): Add origin to transaction confirmation
+- [#10312](https://github.com/MetaMask/metamask-extension/pull/10312): Ensure second click of network menu button closes network menu
```
Comment (Gudahtt) [inline CHANGELOG.md:11]: This entry is not needed. The bug fixed here was not present in v9.0.4.

```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,18 @@
...
+- [#10266](https://github.com/MetaMask/metamask-extension/pull/10266): Update `ko` localized messages
+- [#10263](https://github.com/MetaMask/metamask-extension/pull/10263): Update `id` localized messages
+- [#10347](https://github.com/MetaMask/metamask-extension/pull/10347): Require click of "Continue" button to interact with swap screen if there is a price impact warning for present swap
+- [#10373](https://github.com/MetaMask/metamask-extension/pull/10373): Change copy of submit button on swaps screen
+- [#10346](https://github.com/MetaMask/metamask-extension/pull/10346): Swaps token sources/verification messaging update
+- [#10378](https://github.com/MetaMask/metamask-extension/pull/10378): Stop showing the window.web3 in-appi popup if the dapp is just using web3.currentProvider
```
Comment (Gudahtt) [inline CHANGELOG.md:17]: Typo:
```suggestion
- [#10378](https://github.com/MetaMask/metamask-extension/pull/10378): Stop showing the window.web3 in-app popup if the dapp is just using web3.currentProvider
```

**Review by Gudahtt — CHANGES_REQUESTED** [review]
These two PRs had user-facing changes, but are missing from this changelog:
https://github.com/MetaMask/metamask-extension/pull/10326
https://github.com/MetaMask/metamask-extension/pull/10386

**Comment by danjm** [comment]
@Gudahtt your comments have been addressed

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by brad-decker — APPROVED** [review]
LGTM

**Pull request merged**